### PR TITLE
Attatchment card style

### PR DESCRIPTION
**Pull request by nickbristow**

### CODE REVIEW

**Comment by nickbristow** [comment]
c2

**Comment by jessieay** [comment]
😍

File: app/assets/javascripts/details/app.js [hunk 1]
```diff
@@ -1,3 +1,19 @@
 $(document).ready(function(){
-  // Do something here
+
+  //Attachment functionality
+  //Auto submits form
+  //Shows loading file
+  $(document).on('change','.attachment-file-input',function(){
```
Comment (rememberlenny) [inline app/assets/javascripts/details/app.js]: Can you write some tests around this?

Comment (rememberlenny) [inline app/assets/javascripts/details/app.js]: Heres how the other JS tests are being written: https://github.com/18F/C2/blob/new-details-page/edit-mode/spec/javascripts/state/edit_mode_spec.js.coffee

File: app/controllers/attachments_controller.rb [hunk 2]
```diff
@@ -2,19 +2,18 @@ class AttachmentsController < ApplicationController
   before_action ->{authorize proposal, :can_show!}, only: [:create, :show]
   before_action ->{authorize attachment}, only: [:destroy]
   rescue_from Pundit::NotAuthorizedError, with: :auth_errors
-  respond_to :js, only: [:create, :destroy]
+  respond_to :json, only: [:create, :destroy]
```
Comment (rememberlenny) [inline app/controllers/attachments_controller.rb]: Can you add tests for these endpoints for the attachment stuff?

**Comment by rememberlenny** [comment]
Adding the inline comment here:

Can you write some tests around the JS?

Heres how the other JS tests are being written: https://github.com/18F/C2/blob/new-details-page/edit-mode/spec/javascripts/state/edit_mode_spec.js.coffee

**Comment by nickbristow** [comment]
Will do @rememberlenny

**Comment by rememberlenny** [comment]
@nickbristow Did you want to fix/merge this?

**Comment by nickbristow** [comment]
@rememberlenny trying to write some decent tests and i've had to rework some stuff. Trying to finish COB

**Pull request merged**